docs/source/optimization_toolbox/usage_guides/quantization.mdx

```diff
@@ -73,6 +73,23 @@ empty_model.tie_weights()
 quantized_model = load_quantized_model(empty_model, save_folder=save_folder, device_map="auto")
 ```
 
+### Exllama kernels for faster inference
+
+For 4-bit model, you can use the exllama kernels in order to a faster inference speed. You just need to pass `disable_exllama = False` in [`~optimum.gptq.load_quantized_model`]. In order to use these kernels, you need to have the entire model on gpus.
```

Comment:
Isn't False the default?

Reply:
Yes, the default behavior is [False](url). I will clarify the message.